### PR TITLE
Allow setting hosts in the config

### LINKED ISSUE #26: Support setting default host name via config
As we generate the `config.local.yaml` on `composer chassis init` we could also set the value of `hosts` from the config.

We have this available in `local-server` right now so would be good to maintain feature parity as far as possible.

This could also be used for the default machine name once #13 is ready upstream.

### PR DESCRIPTION
**Pull request by roborourke**
Fixes #26

- [x] Documentation
- [x] Sanitise host names

### CODE REVIEW

File: inc/class-command.php [hunk 1]
```diff
@@ -255,4 +256,30 @@ protected function secure( InputInterface $input, OutputInterface $output ) {
...
 	}
+
+	/**
+	 * Get the name of the project host names from the local config.
+	 *
+	 * @return array
+	 */
+	protected function get_project_hosts() : array {
+
+		$composer_json = json_decode( file_get_contents( getcwd() . '/composer.json' ), true );
```
Comment (rmccue) [inline inc/class-command.php]: Can we use `Altis\get_config()` directly? Think it might be loaded via autoload already 🤔

Comment (roborourke) [reply inc/class-command.php]: Hmm. In not super confident that's the case. I borrowed this from the `local-server` implementation

Comment (roborourke) [reply inc/class-command.php]: Or at least we shouldn't use it unless we make `altis/core` a dependency

Comment (rmccue) [reply inc/class-command.php]: I think we can make it a dep easily enough; the problem with not using it is that we'll potentially have differences between our regular parsing and here; e.g. env-specific configuration doesn't work the same - obviously not a huge concern, but the inconsistency is harmful.

Comment (roborourke) [reply inc/class-command.php]: `Altis/get_config()` isn't defined when the composer command is run. Also this config only really relates to local so I think we have to accept the limitation re. parsing for env specific config.

Comment (rmccue) [reply inc/class-command.php]: Rats, oh well then!

File: docs/README.md [hunk 2]
```diff
@@ -5,6 +5,29 @@
...
+It is possible to configure the host names for your local environment thruogh the `composer.json` file. This should be done before running the setup steps below, otherwise you may need to edit the generated `chassis/config.local.yaml` file directly.
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"local-chassis": {
+					"hosts": [
+						"project.local",
```
Comment (rmccue) [inline docs/README.md]: I wonder whether we should make this a more general option, like `altis.project-name` or something, and derive the hosts from that?

**Pull request merged**